elfi/visualization/visualization.py

```diff
@@ -397,3 +397,49 @@ def plot_discrepancy(gp, parameter_names, axes=None, **kwargs):
         axes[idx].set_axis_off()
 
     return axes
+
+
+def plot_gp(gp, parameter_names, axes=None, resol=50, const=None, **kwargs):
+    """Plot pairwise relationships as a matrix with parameters vs. discrepancy.
+
+    Parameters
```

Comment:
Well, I think it is time to add some [docstring checking](https://github.com/PyCQA/pydocstyle) in Makefile. Otherwise we should skip double documentation in `defining` and `calling` functions.

Reply:
Inheriting docstrings would be great. But until then, explicit is better than implicit. :)